monitors/rootmonitor/include/eudaq/ROOTMonitorWindow.hh

```diff
@@ -0,0 +1,175 @@
+#ifndef EUDAQ_INCLUDED_ROOTMonitorWindow
+#define EUDAQ_INCLUDED_ROOTMonitorWindow
+
+#ifndef __CINT__
+# include "TGFrame.h"
+# include "TGStatusBar.h"
+# include "TGListTree.h"
+# include "TGToolBar.h"
+# include "TContextMenu.h"
+# include "TRootEmbeddedCanvas.h"
+# include "TGButton.h"
+# include "RQ_OBJECT.h"
+#endif
+#include <functional>
+
+class TApplication;
+class TTree;
+namespace eudaq{ class Event; }
+
+/// A placeholder for ROOT monitoring
+class ROOTMonitorWindow : public TGMainFrame {
+  static constexpr const char* NAME = "ROOTMonitorWindow";
+  RQ_OBJECT(NAME)
+public:
+  ROOTMonitorWindow(TApplication*, const std::string& name);
+  ~ROOTMonitorWindow();
+
+  /// List of FSM states
+  enum class Status { idle, configured, running, error };
```

Comment:
EUDAQ2 is already shipped with a set of states - is there a reason not to reuse them?

Reply:
True! I will indeed make use of them in my next push.

Reply:
Addressed in 22654ad8880af49de4fe15f3f5c25eb51c486fb6.